multiversx_sdk/__init__.py

```diff
@@ -65,5 +68,6 @@
     "GenericError", "GenericResponse", "ApiNetworkProvider", "ProxyNetworkProvider",
     "UserSigner", "Mnemonic", "UserSecretKey", "UserPublicKey", "ValidatorSecretKey",
     "ValidatorPublicKey", "UserVerifier", "ValidatorSigner", "ValidatorVerifier", "ValidatorPEM",
-    "UserWallet", "UserPEM", "QueryRunnerAdapter", "TransactionsConverter"
+    "UserWallet", "UserPEM", "QueryRunnerAdapter", "TransactionsConverter", "DelegationTransactionsOutcomeParser",
+    "find_events_by_identifier"
```

Comment:
They seem to be only partially sorted.

Reply:
The `__all__` list is not sorted at all.